gb-cpu/src/memory/zone/rom/mbc.rs

```diff
@@ -0,0 +1,4 @@
+#[derive(Debug)]
+pub enum Mbc {
+    RomOnly,
```

Comment:
No MBC is a type of MBC in the documentation: https://gbdev.io/pandocs/MBCs.html

Think of it as how zero is a number representing no number

Reply:
there's no MBC cartridge only cartridge of MBC**1**, MBC**2*, MBC**3**, MBC**5**, ...

Reply:
I'm not quite sure to get what you mean here, but i think we think alike on this one: If we have an interface that selects between the different MBCs, RomOnly has to be a part of it. 

We can't have an interface for MBCs and on the side an interface for RomOnly, as it adds an unnecesary level of indirection.
This MBC enum is my propostion of this interface, eventually it will go:

```
enum MBC {
  RomOnly,
  MBC1,
  MBC2,
  etc...
}
```

Reply:
It's the name of the enum, has I already said:
> RomOnly don't have a MBC so It's unexpected to have it listed together in a list the list all the MBC controller

What interface are you referencing ?

Reply:
Oh, I got you! i think we could change the name of RomOnly to NoMbc, that would be way clearer.

Or if you have a better name I'm all for it!

Reply:
I thought some way to resolve this conflict:
- rename `enum MBC` to `enum ControllerType` (or something like that)
- rename `RomOnly` to `None/NoMbc`

Reply:
I can settle for "enum Controller" even if I think it's longer and has less information than MBC, as it stands for Memory Bank Controller. But that's fine, it works too

But the fact that it's an enum makes it a type, so we don't need to put type in the name we chose